jdi-light/src/main/java/com/epam/jdi/light/elements/init/entities/collection/EntitiesCollection.java

```diff
@@ -111,18 +111,22 @@ static Object getElement(String name) {
             return getElementInSection(split[1], split[0]);
         if (ELEMENTS.has(name)) {
             List<Object> elements = ELEMENTS.get(name);
-            return elements.size() == 1
-                    ? elements.get(0)
-                    : LinqUtils.first(elements, el -> {
-                WebPage page = ((ICoreElement) el).base().getPage();
-                return page != null && page.getName().equals(getCurrentPage());
-            });
+            if (elements.size() > 1) {
+                Object element = LinqUtils.first(elements, el -> {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'LinqUtils.first' due to existing static import 'com.epam.jdi.tools.LinqUtils.*'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5217753)